### PR TITLE
Rebased: Setting up ActiveJob (with DJ backend) to check DUNS

### PR DESCRIPTION
**Pull request by adelevie**
This would negate: https://github.com/18F/micropurchase/pull/440

- Sets up ActiveJob/Delayed job
- Delays a call to the `SamReckoner`

Fixes https://github.com/18F/micropurchase/issues/381

### CODE REVIEW

File: features/step_definitions/page_steps.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Then(/^I refresh the page$/) do
```
Comment (jessieay) [inline features/step_definitions/page_steps.rb]: I think this should be a `When` rather than `Then`

**Comment by jessieay** [comment]
this looks great to me! wanna try pushing it to staging manually before we merge to `develop` ?

**Comment by adelevie** [comment]
Sure thing. We're fixing the env var situation first so new deploys have access to the right API keys.

**Comment by adelevie** [comment]
Ready after:
- https://github.com/18F/micropurchase/pull/448 is merged
- this branch is rebased

**Comment by jessieay** [comment]
this now has a merge conflict @adelevie

**Comment by adelevie** [comment]
rebased

**Pull request merged**